screens/ChooseStyleScreen.tsx

```diff
@@ -18,16 +19,29 @@ type Props = NativeStackScreenProps<RootStackParamList, 'ChooseStyleScreen'>
 
 export default function ChooseStyleScreen({ route, navigation }: Props) {
   const [projectTheme, setProjectTheme] = useState<IProjectTheme>(ProjectThemes[0] || null)
-
+  const [loading, setLoading] = useState(false)
+  const [snackbarOpen, setSnackbarOpen] = useState(false)
+  const [snackbarMessage, setSnackbarMessage] = useState('')
   const { image } = route.params
 
-  const handleContinue = () => navigation.navigate('FinishedArtScreen', { image: image })
+  const handleContinue = () => {
+    setLoading(true)
+    navigation.navigate('FinishedArtScreen', { image: image })
+  }
 
   return (
     <View style={styles.container}>
+      <SnackBar
```

Comment:
I added a snackbar for displaying errors when posting to the backend, maybe lex can use it

Reply:
![image](https://user-images.githubusercontent.com/43485369/155996439-ac6cbf36-4472-48e9-be61-55d9b373265c.png)

Reply:
thanks, ill try it again i guess